ablySpec/RealtimeClient.connection.swift

```diff
@@ -713,6 +713,56 @@ class RealtimeClientConnection: QuickSpec {
                 }
 
             }
+
+            // RTN12
+            context("close") {
+
+                // RTN12b
+                it("should receive CLOSED action within the default request timeout") {
+                    let options = AblyTests.commonAppSetup()
+                    options.autoConnect = false
+                    let client = ARTRealtime(options: options)
+                    client.setTransportClass(TestProxyTransport.self)
+                    client.connect()
+
+                    let transport = client.transport as! TestProxyTransport
+                    transport.actionsIgnored += [.Closed]
+
+                    var states: [ARTRealtimeConnectionState] = []
+                    var start: NSDate?
+                    var end: NSDate?
+
+                    client.eventEmitter.on { state, errorInfo in
+                        switch state {
+                        case .Connected:
+                            client.close()
+                        case .Closing:
+                            start = NSDate()
+                            states += [state]
+                        case .Closed:
+                            end = NSDate()
+                            states += [state]
+                        case .Disconnected:
```

Comment:
It should not happen but I want to catch it and check if `states` doesn't have a `.Disconnected` event.